examples/sbol2/CreLoxRecombination.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Representing Cre-Lox Recombination with SBOL2\n",
+    "\n",
+    "Cre-Lox recombination is a site-specific recombination event, often used to modify sequences in synthetic biology by flipping or excising segments of DNA. This can be used to activate or deactivate genes or other elements.\n",
+    "\n",
+    "In this example, we will represent a situation where an unknown coding sequence (CDS) is initially in the wrong orientation and flanked by two loxP recombination sites. Upon recombination, the CDS will be flipped into the correct orientation. We'll use SBOL2 concepts like `Component`, `SequenceAnnotation`, `Range`, `GenericLocation`, and `SequenceConstraint` to model this event."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sbol2\n",
```

Comment:
This is a giant block of code. I would suggest breaking it into chunks with key code comments moving to instead by notebook text.

Reply:
Good point. Addressed in latest commit.